core/src/main/java/org/polypheny/db/partition/properties/PartitionProperty.java

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
This class is used in the Catalog. Since the Catalog should be as capsuled as possible, why not moving it to the catalog package? Is there a specific reason?

Furthermore, since the class is used in the catalog, it must be immutable.